### ISSUE TITLE
Imgur plugin unusable

### ISSUE THREAD

**Issue by Tailszefox** [olav-st/screencloud-plugins#19]
I already posted about this issue [on the main repository](https://github.com/olav-st/screencloud/issues/154), but since this is about a plugin, it's probably better to post about it here.

Currently running Debian Jessie, I'm trying to use the Imgur plugin with the latest version of Screencloud installed from the repository. However, I'm getting the following errors each time I start Screencloud:

```
Error in file: /home/tails/.local/share/data/screencloud/ScreenCloud/plugins/imgur/main.py
Traceback (most recent call last):
  File "<string>", line 5, in <module>
  File "/home/tails/.local/share/data/screencloud/ScreenCloud/plugins/imgur/modules/pyimgur/__init__.py", line 39, in <module>
    import requests
  File "/home/tails/.local/share/data/screencloud/ScreenCloud/plugins/imgur/modules/requests/__init__.py", line 53, in <module>
    from .packages.urllib3.contrib import pyopenssl
  File "/home/tails/.local/share/data/screencloud/ScreenCloud/plugins/imgur/modules/requests/packages/urllib3/contrib/pyopenssl.py", line 69, in <module>
    ssl.PROTOCOL_SSLv3: OpenSSL.SSL.SSLv3_METHOD,
AttributeError: 'module' object has no attribute 'PROTOCOL_SSLv3'
```

followed by
```
Failed to call ImgurUploader.isConfigured()
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: cannot import name ImgurUploader
Traceback (most recent call last):
  File "<string>", line 1, in <module>
NameError: name 'ImgurUploader' is not defined
```

And when trying to configure the plugin:
```
Failed to call ImgurUploader.showSettingsUI()
Traceback (most recent call last):
  File "<string>", line 1, in <module>
NameError: name 'imgur_u' is not defined
```

I posted a solution to this issue [here](https://github.com/olav-st/screencloud/issues/154#issuecomment-127281742). Basically, this is due to the dependencies being out of date. Simply bumping the submodules to the latest version available should do the trick.

**Comment by Pharmasolin** [comment]
Please fix some one this

**Comment by Pharmasolin** [comment]
Warning: Permanently added the RSA host key for IP address '192.30.252.130' to the list of known hosts.
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

i can't do this as you wrote

**Comment by Tailszefox** [comment]
You probably didn't configure your SSH access. I've changed it to use HTTPS instead, so for step 2, use this: `git clone https://github.com/olav-st/screencloud-plugins.git`

**Comment by Pharmasolin** [comment]
@Tailszefox thx for help now it works! I updated this bug report to those who upgraded from 15.04 to 15.10 and received this issue.

https://github.com/olav-st/screencloud/issues/154#issuecomment-127281742

**Comment by Tailszefox** [comment]
Fixed for me with 04f561680ffce3d96e8149a0b7bcf93eb309034a , thanks!
